viam-cartographer.go

```diff
@@ -116,6 +180,12 @@ func New(
 		return nil, err
 	}
 
+	if !modularizationV2Enabled {
+		if err := vcConfig.SetupDirectories(svcConfig.DataDirectory, logger); err != nil {
```

Comment:
We can't do this if the feature flag is enabled, or we will trigger the `VIAM_CARTO_DATA_DIR_INVALID_DEPRECATED_STRUCTURE` error